.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -3,80 +3,76 @@ on:
   push:
     branches:
       - master
-      - "1.4.x"
```

Comment:
Why did you remove the 1.4.x and 1.5.x branches?

Reply:
Removing old stuff we're not targeting.

Reply:
I assume that those branches still need to be maintained for customer support. I suppose that they could be maintained in the repo in the DataStax org. Who is going to be handling bug fixes for customer support. I would expect over time that it will be us. If that is the case, does it make sense to keep them?

@bradfordcp can you chime in here?

Reply:
Wasn't that the deal that old versions are going to stay in the datastax/cass-operator?

Reply:
We can add back or update needed functionality in follow up PRs. I don't want to hold up merging on this as it currently does not impact k8ssandra.